schema/properties_directory/non-scalar/averaged_potential.json

```diff
@@ -0,0 +1,15 @@
+{
+    "schemaId": "properties-directory/non-scalar/averaged-potential",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "averaged potential schema",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "macroscopicMinima": {
```

Comment:
Should we have the value at the cell edge here too?

Reply:
ok yeah, good idea. Is it always the first and last value of the average.x output?

Reply:
Perhaps it is. Maybe we should calculate an average over some small region around the edges to be safe?